packages/ses/SECURITY.md

```diff
@@ -10,16 +10,34 @@ be made for the most recent branch only.
 | ------- | ------------------ |
 | 0.12.x  | :white_check_mark: |
 
-## Reporting a Vulnerability
-
-Please help us practice coordinated security bug disclosure. If you find a
-security-sensitive bug that should not be revealed publically until a fix is
-available, please send email to `security` at (@) `agoric.com`. To encrypt,
-please use my (@warner) personal GPG key [A476E2E6 11880C98 5B3C3A39 0386E81B
-11CAA07A](http://www.lothar.com/warner-gpg.html) . Keybase users can also
-send messages to `@agoric_security`, or share code and other log files via
-the Keybase encrypted file system
-(`/keybase/private/agoric_security,$YOURNAME`).
-
-We will create a GitHub security advisory promptly. Let us know your GitHub
-username and we'll add you to the collaborator list for further discussion.
+
+## Coordinated Vulnerability Disclosure of Security Bugs
+
+At Agoric, we believe that strong security requires strong collaboration with security researchers. If you believe that you have found a security sensitive bug in that should not be disclosed until a fix has been made available, we encourage you to report it. To report a bug in HardenedJS, you have several options that include: 
```

Comment:
> a security sensitive bug in that should not be disclosed until 

Extra `in` ?

Reply:
Thanks; fixed this and included language about SES and fearless cooperation.